docker-mender-convert

```diff
@@ -16,8 +16,8 @@
 
 set -e
 
-GIT_TAG_OR_BRANCH_NAME="$(git describe --exact HEAD 2>/dev/null || \
-    git for-each-ref "refs/heads/*" --format '%(refname:short)' --points-at HEAD 2>/dev/null)"
+GIT_TAG_OR_BRANCH_NAME="$(git describe --tags --exact-match 2>/dev/null || \
+    git branch --show current 2>/dev/null)"
```

Comment:
The downside of this fallback (`git branch` instead of `git for-each-ref "refs/heads/*"`) is that it won't work on detached HEAD - which is what I suspect happens in CI and motivated this piece of code in the place.

What about leaving it as it was but cutting the output to one line?
```suggestion
GIT_TAG_OR_BRANCH_NAME="$(git describe --exact HEAD 2>/dev/null || \
    git for-each-ref "refs/heads/*" --format '%(refname:short)' --points-at HEAD 2>/dev/null | head -n1)"
```

It is still imperfect, because it will pick the first branch in alphanumerical order always (a Git branch has no date information so it cannot be sorted by date, for example). But at least it won't break as you have experienced.